packages/@okta/courage-for-signin-widget/README.md

```diff
@@ -0,0 +1,11 @@
+# @okta/courage-for-signin-widget
+
+Generates a custom version of Courage for the Sign-In Widget. 
+It's intend for internal developers and please open a ticket if any questions.
+
+## Steps
+
+0. make sure VPN is connected
+1. upgrade courage version. e.g. `yarn upgrade @okta/courage@4.4.0-beta.xxxxxx`
```

Comment:
let's change this to use release versions. It is my understanding we should try to use release versions of i18n and courage in the SignIn Widget, and use beta versions internally only
```
yarn upgrade @okta/courage@4.4.0.xxxxx
```

Reply:
(replied below)